src/README.md

```diff
@@ -179,6 +179,12 @@ class Routine {
 | voltageMinDesign | number | Desired minimum output voltage |
 | voltageNow | number | Battery's voltage (V) |
 
+### AcPowerRoutineParams
+| Property Name | Type | Description |
+------------ | ------- | ----------- |
+| expectedStatus | string | The expected status of the AC ('connected', 'disconnected' or 'unknown') |
+| expectedPowerType | string | (Optional) If specified, this must match the type of power supply for the routine to succeed. |
```

Comment:
To be consistent, let's use the same notion as in "RoutineStatus". That is, follow "expectedPowerType" with an * --> "expectedPowerType*" and remove "(Optional)"